config/peerpods/podvm/lib.sh

```diff
@@ -187,6 +187,55 @@ function prepare_source_code() {
     fi
 }
 
+# Download and extract pause container image
+# Accepts three arguments:
+# 1. pause_image_repo_url: The registry URL of the OCP pause image.
+# 2. pause_image_tag: The tag of the OCP pause image.
+# 2. auth_json_file (optional): Path to the registry secret file to use for downloading the image
+function download_and_extract_pause_image() {
+
+    # Set default values for the OCP pause image
+    # This image is multi-arch
+    PAUSE_IMAGE_REPO_DEFAULT="quay.io/openshift-release-dev/ocp-v4.0-art-dev@sha256"
```

Comment:
nit: maybe define the defaults in the top of the file?

Reply:
yeah, I thought about it but then not sure what made me put here ;) .. I think I was in two minds whether to put it at the top or in the function..